modules/ml/src/main/java/org/apache/ignite/ml/dataset/impl/cache/util/ComputeUtils.java

```diff
@@ -183,6 +199,15 @@
         });
     }
 
+    private static <K, V> void setTransformationSeedForPartition(UpstreamTransformerChain<K, V> transformersChain,
```

Comment:
What's going on? We really need this method here?

Reply:
Refactored with `UpstreamTransformerChain::modifySeed`